chart/values.schema.json

```diff
@@ -132,7 +132,7 @@
                             "default": "/"
                         },
                         "pathType": {
-                            "description": "The pathType for the web Ingress (required for Kubernetes 1.19 and above).",
+                            "description": "The pathType for the web Ingress (required for Kubernetes 1.20 and above).",
```

Comment:
We should probably leave these as-is, or remove them altogether.

Reply:
ok I revert my change